content/en/docs/ops/integrations/loadbalancers/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Third Party Load Balancers
+description: How to integrate Istio with third party load balancers.
+weight: 90
+keywords: [traffic-management,ingress]
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+Istio provides both an ingress and service mesh implementation, which can be used together or separately. While these are designed to work together seamlessly, there are times when integrating with a third party ingress is required. This could be for migration purposes, feature requirements, or personal preferences.
+
+## Integration Modes
+
+In "standalone" mode, the third party ingress is directly sending to backends.
+In this case, the backends presumably have Istio sidecars injected;
+
+{{< mermaid >}}
+graph LR
+    cc((Client))
+    tpi(Third Party Ingress)
+    a(Backend)
+    cc-->tpi-->a
+{{< /mermaid >}}
+
+In this mode, things mostly just work.
+Clients in a service mesh do not need to be aware that the backed they are connecting to have a sidecar.
+However, the ingress will not use mTLS, which may lead to undesirable behavior.
+As a result, most of the configuration for this setup is around enabling mTLS.
+
+In "chained" mode, we use both the third party ingress *and* Istio's own Gateway in parallel.
```

Comment:
I'm not sure I like the word `parallel` here, as it means something different in a circuit and are really in `series`. Not sure I have a bite wording, and maybe just leave it off:
```suggestion
In "chained" mode, we use both the third party ingress *and* Istio's own Gateway.
```

Reply:
Maybe `in parallel` -> `in sequence` ?